reactor-netty5-core/src/test/java/reactor/netty5/NettyOutboundTest.java

```diff
@@ -255,6 +256,7 @@ void sendFileWithForceChunkedFileUsesStrategyChunks()
 					protected void encodeAndClose(ChannelHandlerContext ctx, Buffer msg,
 							List<Object> out) {
 						out.add(msg.readCharSequence(msg.readableBytes(), StandardCharsets.UTF_8));
+						msg.close();
```

Comment:
What do you think of overriding `encode()` method, thus `encodeAndClose()` will close the message for us?